### DISCUSSION TITLE
[Question] How to get a street names in flet map

### DISCUSSION THREAD

**Discussion by Siva-Rajan-R** [flet-dev/flet#4182] [Q&A] [answered]
### Question

When i give a latitude and longitude how to get a street name in flet map is there possible,or i have to use a external api ?

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment, marked as answer]
To display maps we make use of [flutter-map](https://docs.fleaflet.dev/). Feel free to search if it makes it possible. 
I haven't seen that yet.
